backend/docker.go

```diff
@@ -304,6 +300,8 @@ func buildDockerImageSelector(selectorType string, client *docker.Client, cfg *c
 	switch selectorType {
 	case "tag":
 		return &dockerTagImageSelector{client: client}, nil
+	case "env":
+		return image.NewEnvSelector(cfg)
```

Comment:
Wait, like, you weren't kidding about this already existing!

So how does this play out for which dockerTag to snag? Or is it by another means entirely? I'm guessing this will require some updates to the worker config and install script?

Reply:
It's completely separate from docker tags, yep!  I would like to include some env-based selection docs in this PR, too.  Once upon a time, I thought we had some docs about it, but I don't see any in the README, so fixing!